plugins/node/opentelemetry-instrumentation-aws-lambda/src/aws-lambda.ts

```diff
@@ -120,8 +120,8 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
       const span = plugin.tracer.startSpan(name, {
         kind: SpanKind.SERVER,
         attributes: {
-          [FaasAttribute.FAAS_EXECUTION]: context.awsRequestId,
-          [FaasAttribute.FAAS_ID]: context.invokedFunctionArn,
+          [SemanticAttributes.FAAS_EXECUTION]: context.awsRequestId,
+          'faas.id': context.invokedFunctionArn,
```

Comment:
faas.id is used as a span attribute here, but should be a resource attribute. I left it here to make this PR as small as possible while passing and changing no behavior. I will create a follow up bug to fix this.